src/slack/constants.js

```diff
@@ -130,3 +132,22 @@ export const ORDER_NOTE_ACTIONS = vlConstants.ORDER_NOTE_ACTIONS
 export const ORDER_COMPANY_ACTIONS = vlConstants.ORDER_COMPANY_ACTIONS
 
 export const DELIVERY_PLACE_ACTIONS = vlConstants.DELIVERY_PLACE_ACTIONS
+
+export const ACTION_RANKS = {
```

Comment:
- `is_personal` and `is_company` are king of a `type` action. While I set `order.step` type value, I could not predict is it a `is_personal` or `is_company`. 
- `note` is previous action of `note-yes` and `note-no`
- `urgent` and `company` are on the same level/rank in **process tree**.

Reply:
right, makes sense. anyway, maybe we should remove `note_yes` and move the numbering - the `note-yes` is not really a state, it's basically the `reason` step, right? and as I mentioned in other comment - it seems like we don't need to set the `note_yes` step at all, it will get overwritten shortly